From 56efeab366da412ee4196107c08174f32ed83c9a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=A9r=C3=A9mie=20Galarneau?=
 <jeremie.galarneau@efficios.com>
Date: Mon, 13 Nov 2017 10:31:29 -0500
Subject: [PATCH] Fix warning: src/bin/lttng/utils.c: cast incompatible pointer
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Reported-by: Philippe Proulx <eeppeliteloop@gmail.com>
[Philippe: backport from upstream commit 56efeab3]
Signed-off-by: Jérémie Galarneau <jeremie.galarneau@efficios.com>
---
 src/bin/lttng/utils.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/bin/lttng/utils.c b/src/bin/lttng/utils.c
index 885f498b..52a2440e 100644
--- a/src/bin/lttng/utils.c
+++ b/src/bin/lttng/utils.c
@@ -387,7 +387,7 @@ int check_relayd(void)
 	 * A successful connect means the relayd exists thus returning 0 else a
 	 * negative value means it does NOT exists.
 	 */
-	ret = connect(fd, &sin, sizeof(sin));
+	ret = connect(fd, (struct sockaddr *) &sin, sizeof(sin));
 	if (ret < 0) {
 		/* Not found. */
 		ret = 0;
--
2.15.0

